<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>665</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>665</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ld-analyse</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ld-analyse-resources.qrc">
    <normaloff>:/icons/Graphics/ld-analyse logo 128px.png</normaloff>:/icons/Graphics/ld-analyse logo 128px.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>400</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>210</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QSpinBox" name="frameNumberSpinBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="maximum">
        <number>80000</number>
       </property>
       <property name="value">
        <number>79999</number>
       </property>
      </widget>
      <widget class="QPushButton" name="previousPushButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Prev</string>
       </property>
      </widget>
      <widget class="QPushButton" name="nextPushButton">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>10</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="highlightDropOutsCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Highlight dropouts</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="showActiveVideoCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Show active video extent</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="firstFieldGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>221</width>
         <height>211</height>
        </rect>
       </property>
       <property name="title">
        <string>First field:</string>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Field #:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="firstFieldLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Sync Conf.:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Dropouts:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>DO Length:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>VBI:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>16:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>17:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>18:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="even0VbiLabel">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>140</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>No metadata</string>
        </property>
       </widget>
       <widget class="QLabel" name="even1VbiLabel">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>160</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>No metadata</string>
        </property>
       </widget>
       <widget class="QLabel" name="even2VbiLabel">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>180</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>No metadata</string>
        </property>
       </widget>
       <widget class="QLabel" name="firstFieldSyncConfLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
       <widget class="QLabel" name="firstFieldDropoutsLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>No metadata</string>
        </property>
       </widget>
       <widget class="QLabel" name="firstFieldDoLengthLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>No metadata</string>
        </property>
       </widget>
      </widget>
      <widget class="QRadioButton" name="sourceRadioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Source</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="combFilterRadioButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Comb filter</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="secondFieldGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>380</y>
         <width>221</width>
         <height>211</height>
        </rect>
       </property>
       <property name="title">
        <string>Second field:</string>
       </property>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Field #:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="secondFieldLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Sync Conf.:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Dropouts:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>DO Length:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>VBI:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>16:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>17:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>18:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="odd0VbiLabel">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>140</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>No metadata</string>
        </property>
       </widget>
       <widget class="QLabel" name="odd1VbiLabel">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>160</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>No metadata</string>
        </property>
       </widget>
       <widget class="QLabel" name="odd2VbiLabel">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>180</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>No metadata</string>
        </property>
       </widget>
       <widget class="QLabel" name="secondFieldSyncConfLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
       <widget class="QLabel" name="secondFieldDropoutsLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>No metadata</string>
        </property>
       </widget>
       <widget class="QLabel" name="secondFieldDoLengthLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>No metadata</string>
        </property>
       </widget>
      </widget>
      <widget class="QCheckBox" name="reverseFieldOrderCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Reverse still-frame field order</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="FrameQLabel" name="frameViewerLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Graphics Label</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="frameHorizontalSlider">
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1234</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_TBC_file"/>
    <addaction name="separator"/>
    <addaction name="actionSave_frame_as_PNG"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionLine_scope"/>
    <addaction name="actionVBI"/>
    <addaction name="actionNTSC"/>
    <addaction name="actionVideo_metadata"/>
    <addaction name="actionVITS_Metrics"/>
    <addaction name="actionDropout_analysis"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_ld_analyse"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action1_1_Frame_size"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout_ld_analyse">
   <property name="text">
    <string>About ld-analyse...</string>
   </property>
  </action>
  <action name="actionOpen_TBC_file">
   <property name="text">
    <string>Open TBC file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionLine_scope">
   <property name="text">
    <string>Line scope...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionVBI">
   <property name="text">
    <string>VBI...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionNTSC">
   <property name="text">
    <string>NTSC...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionVideo_metadata">
   <property name="text">
    <string>Video metadata...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action1_1_Frame_size">
   <property name="text">
    <string>1:1 Frame size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionDropout_analysis">
   <property name="text">
    <string>Dropout analysis...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSave_frame_as_PNG">
   <property name="text">
    <string>Save frame as PNG...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionVITS_Metrics">
   <property name="text">
    <string>VITS Metrics...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FrameQLabel</class>
   <extends>QLabel</extends>
   <header>frameqlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ld-analyse-resources.qrc"/>
 </resources>
 <connections/>
</ui>
